lib/iris/fileformats/pp.py

```diff
@@ -46,13 +46,19 @@
 
 try:
     import mo_pack
-except ImportError:
-    mo_pack = None
+except ImportError as err:
+    if err.message.startswith('No module named'):
+        mo_pack = None
+    else:
+        raise
 
 try:
     from iris.fileformats import pp_packing
-except ImportError:
-    pp_packing = None
+except ImportError as err:
+    if err.message.startswith('No module named'):
```

Comment:
You won't ever get this error. If Iris has been installed without `pp_packing` then the error will be "cannot import name ...". As it stands your code is treating this as an error, which it definitely isn't.